noxfile.py

```diff
@@ -12,7 +12,7 @@
 
 def deps(session):
     session.install("--upgrade", "setuptools", "pip")
-    session.install("-r", "requirements/dev.txt", ".", silent=True)
+    session.install("-r", "requirements/dev.txt", "-e", ".", silent=True)
```

Comment:
@omry This fixes a nox failure on CircleCI. My understanding of the problem was that without `-e`, pip would install the package (with properly generated grammar files) somewhere else on the system, but would actually run with the local code due to `"."` being the first path being searched. And this local code doesn't have the grammar files => boom.

Just wanted to mention it in case you'd have another suggestion to avoid this problem (it doesn't seem to happen with Hydra and I'm not sure why -- I haven't looked closely at Hydra's noxfile)

Reply:
Check the sdist and make sure the the generated grammar is included. make sure you are running the antlr command as a part of the sdist command. Check the setup.py in Hydra.

Reply:
Yes I'm doing that. It's what I meant by "with properly generated grammar files": pip does build the wheel with the grammar files. However since it was installing in a different folder these files were not present in the code actually being used by nox.

Reply:
pip install is expected to install in a different directory, this is normal.

Reply:
Make sure that the files in the sdist are int he right location.

Reply:
> pip install is expected to install in a different directory, this is normal.

I realize that. As example, imagine you're sitting in your git repository and run: `pip install -r requirements/dev.txt .`. Then edit `omegaconf/__init__.py` and add an `assert False` at the top. Then running `python-c "import omegaconf"` will fail, while if you change to a different folder it will succeed. This shows that when you're running in your repo root you're not actually using the codebase installed by `pip install`. In my case this caused crashes because the grammar is only generated there.

Reply:
Juste a quick update: I looked at it a bit more closely and there's some stuff I hadn't understood properly. I'll have a better fix later.

Reply:
So the better fix was supposed to be f1a4d4bf6f0bda6853f49acbe73ae062f8577536 but although it works for `pytest`, it doesn't play way with `coverage`. I found a workaround but actually I was wondering if maybe it'd be better to keep the `-e` flag to `pip install` for the coverage tests, so as to be sure we also check the coverage of files that may not end up being copied by `pip install`. Thoughts?

Reply:
So I went with the `pip install -e` approach for coverage tests as it's the easiest fix (at the same time I made it explicit whether each nox session would use a local or system installation). See cbc098818988aaca830707df5014bbe04d37c5ce

Reply:
This thread of discussion is continued in https://github.com/omry/omegaconf/pull/321#discussion_r513670942